6-Typescript/server/tests/env/mockData.ts

```diff
@@ -0,0 +1,21 @@
+import { ITodo } from "../../interfaces/interfaces";
+import { v4 as uuidv4 } from "uuid";
+import chance from "chance";
+
+export const createTodo = (defParams: Partial<ITodo> = {}): ITodo => {
+    return {
+        todoText: defParams.todoText || `test${chance().integer()}`,
```

Comment:
if the type of `defParams` is `Partial<ITodo>` and can be even `{}`, you can try to access undefined items.
I would use `defParams?.todoId || createDefaultTodoId()`